README.md

```diff
@@ -1,2 +1,50 @@
 # AIPoweredChatbot-G68-PS23
 AI powered chatbot with interactive UI
+# NMCG CHATBOT
+A Chatbot application for Ganga Cleaning Awareness Campaign.
+# Visit our webpage at: 
+https://nmcg-chatbot.onrender.com
+
+
+# Problem Statement
+Create an AI, ML & Chatbot powered interactive mascot (Chacha Chaudhary) and 
+digital avatar to raise awareness and disseminate information about the Namami 
+Gange Programme to school children, the common man, and all stakeholders of 
+NMCG. 
+- The digital avatar/mascot should be able to interact with people of all ages and 
+backgrounds. 
+- It will be deployed at the entrance of the River Basin War Room & Ganga Museum, as 
+well as on the NMCG website. 
+- The goal of the project is to create a fun and engaging way for people to learn about 
+the Namami Gange Programme and the importance of river conservation.
+
+# Getting Started
+  ####  Technology Used
+- HTML
+- CSS
+- SASS
+- BOOTSTRAP 5
+- PYTHON
+- JAVASCRIPT
+- ADOBE ANIMATE
+- GOOGLE API'S
+ ####  Installation
+* There is no need to install this Application.You just need to follow through our Website.
```

Comment:
Add steps to install your code and run the website locally on a system. Website is deployed and available for public access, well and good, but provide the project installation and run instructions as well.